tnb/base_client.py

```diff
@@ -62,28 +62,28 @@ def fetch_multiple_page(self, resource, **kwargs):
 
         return complete_response
 
-    def delete(self, resource, **kwargs):
+    def delete(self, resource: str, **kwargs: object) -> object:
         """
         DELETE a `resource` from a Node
         Return response as Python object
         """
         return self.send_request("DELETE", resource, **kwargs)
 
-    def patch(self, resource, body, **kwargs):
+    def patch(self, resource: str, body: dict, **kwargs: object) -> object:
```

Comment:
The parameter `kwargs` is a `dict` instead of an `object`.

This method returns either `dict` or a `list`.

**Note**: 
- The method documentation is also incorrect. 
- For this change you will also need to import `Union` with: 
```python
from typing import Union
```

```suggestion
    def patch(self, resource: str, body: dict, **kwargs: dict) -> Union[dict, list]:
```